apps/server/src/tasks/SendTestUserEmail.ts

```diff
@@ -0,0 +1,31 @@
+import { fixedGroups } from '@raise/shared';
+import { sendEmail } from '../helpers/email';
+import newUser from '../helpers/email/newUser';
```

Comment:
GitHub's error here isn't very useful, but I think has the right sentiment: this file doesn't seem to exist.